pkg/sql/logictest/testdata/logic_test/create_statements

```diff
@@ -1042,6 +1042,23 @@ CREATE TABLE crdb_internal.session_variables (
    value STRING NOT NULL,
    hidden BOOL NOT NULL
 )  {}  {}
+CREATE TABLE crdb_internal.statement_statistics (
+   aggregated_ts TIMESTAMPTZ NOT NULL,
+   fingerprint_id BYTES NOT NULL,
+   plan_hash INT8 NOT NULL,
+   app_name STRING NOT NULL,
+   metadata JSONB NOT NULL,
+   statistics JSONB NOT NULL,
+   plan JSONB NOT NULL
+)  CREATE TABLE crdb_internal.statement_statistics (
+   aggregated_ts TIMESTAMPTZ NOT NULL,
+   fingerprint_id BYTES NOT NULL,
+   plan_hash INT8 NOT NULL,
+   app_name STRING NOT NULL,
+   metadata JSONB NOT NULL,
+   statistics JSONB NOT NULL,
+   plan JSONB NOT NULL
```

Comment:
Can we remove the plan for now? Since we have a plan_hash which doesn't correlate to to this plan, it might cause confusion. Users might assume a change in the hash indicates a change in the plan which wouldn't always be the case.

Reply:
This view will also be used to serve the DB Console. So this column kind of has to here in order for that to happen. I think what we can do here is rename the current virtual table to something like `crdb_internal.<some_name_suggests_it_should_not_be_used_by_end_user` and we create a `crdb_internal` view on top of it to expose the schema that we want user to query. 

Do you have any thoughts on the name we should use? I'm thinking something like `crdb_internal.private_statement_statistics` or something like that.

Reply:
Discussed offline and renamed the column to `sampled_plan`.